Tests/KSCrash/KSJSONCodec_Tests.m

```diff
@@ -1370,37 +1385,13 @@ - (void) testDeserializeObjectWithNullIgnoreNullInArray
     XCTAssertEqualObjects(value, expected, @"");
 }
 
-- (void) testDeserializeObjectWithNullIgnoreNullInObject
-{
-    NSError* error = (NSError*)self;
-    NSString* json = @"{\"blah\":null}";
-    NSArray* result = [BSG_KSJSONCodec decode:toData(json)
-                                  options:BSG_KSJSONDecodeOptionIgnoreNullInObject
-                                    error:&error];
-    XCTAssertNotNil(result, @"");
-    XCTAssertNil(error, @"");
-    XCTAssertTrue([result count] == 0, @"");
-}
-
-- (void) testDeserializeObjectWithNullIgnoreAllNulls
-{
-    NSError* error = (NSError*)self;
-    NSString* json = @"{\"blah\":null}";
-    NSArray* result = [BSG_KSJSONCodec decode:toData(json)
-                                  options:BSG_KSJSONDecodeOptionIgnoreAllNulls
-                                    error:&error];
-    XCTAssertNotNil(result, @"");
-    XCTAssertNil(error, @"");
-    XCTAssertTrue([result count] == 0, @"");
-}
-
 - (void) testFloatParsingDoesntOverflow
 {
     NSError *error = (NSError*)self;
 
     char * buffer = malloc(0x1000000);
     for (int i = 0; i < 0x1000000; i++) {
-        buffer[i] = ';';
+        buffer[i] = ' ';
```

Comment:
Is this related to the code changes? Just intrigued...!

Reply:
yeah - this wasn't a reasonable test before since having endless semicolons at the end of the file wasn't valid. Using some whitespace keeps the intent (ensuring a float value doesn't overflow into later characters) without causing other parsing errors.